src/test/java/uk/ac/ebi/eva/utils/JobParameterBuilder.java

```diff
@@ -0,0 +1,69 @@
+package uk.ac.ebi.eva.utils;
+
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JobParameterBuilder {
```

Comment:
This class doesn't seem to be used at all, is it from the tests that are under development?

If so, could it be renamed to something that differentiates it from Spring Batch `JobParametersBuilder`? Something like `EvaJobParametersBuilder` could be enough. It could also extend `JobParametersBuilder` class with the EVA-specific methods.